docs/spec.md

```diff
@@ -5295,7 +5659,7 @@ More formally, for all `result_index` in the index space of `results[0]`:
 * (C3) All tensors in `inputs` and `results` have the same shape.
 * (C4) `-R` $\le$ `dimension` $\lt$ `R`, where `R` is rank of `inputs[0]`.
 * (C5) `comparator` has type
-  `(tensor<E1>, tensor<E1>, ..., tensor<EN-1>, tensor<EN-1>) -> tensor<i1>`,
+  `(tensor<E1>, tensor<E1>, ..., tensor<EN-1>, tensor<EN-1>) -> tensor<i1>`.
```

Comment:
I don't think this punctuation computes, given that there's a "where" on the next line?